overcooked_ai_py/agents/agent.py

```diff
@@ -301,28 +309,31 @@ def action(self, state):
         self.env.state = initial_env_state
 
         first_joint_action = first_s_a[0][0]
-        if self.logging_level >= 1: 
+        if self.logging_level >= 1:
             print("expected joint action", first_joint_action)
         action = first_joint_action[self.agent_index]
         return action, {}
 
 
 class GreedyHumanModel(Agent):
     """
+    This is Micah's GreedyHumanModel, which is slightly different to Paul's (called _pk)
+
```

Comment:
We should probably remove comments referring to our names specifically (as we expect that people that don't know us will be trying to use this code), and possibly find better naming conventions instead that capture the essence of the difference between the model types.